inc/dbmysql.class.php

```diff
@@ -874,6 +896,11 @@ function __construct ($dbconnexion, $table, $crit="", $debug=false) {
 
 
    private static function quoteName($name) {
```

Comment:
From my point of view, whatever the visibility, methods should be documented, it can be usefull for developers that would use it or just try to understand ;)
Well for instance this is not (yet? :p) a good example since this is a "simple" method but...